alpine/alpine-unfixed/alpine_test.go

```diff
@@ -0,0 +1,131 @@
+				"testdata/outfiles/alpine-unfixed/3.14/community/freeimage.json": "testdata/golden/3.14/community/freeimage.json",
+				"testdata/outfiles/alpine-unfixed/3.14/main/wget.json":           "testdata/golden/3.14/main/wget.json",
+				"testdata/outfiles/alpine-unfixed/3.14/main/cairo.json":          "testdata/golden/3.14/main/cairo.json",
+				"testdata/outfiles/alpine-unfixed/3.12/main/wget.json":           "testdata/golden/3.12/main/wget.json",
+				"testdata/outfiles/alpine-unfixed/3.12/main/cairo.json":          "testdata/golden/3.12/main/cairo.json",
+				"testdata/outfiles/alpine-unfixed/3.13/main/wget.json":           "testdata/golden/3.13/main/wget.json",
+				"testdata/outfiles/alpine-unfixed/3.13/main/cairo.json":          "testdata/golden/3.13/main/cairo.json",
+			}},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			home := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+				if fileName, ok := tt.fields.homeFiles[r.URL.Path]; !ok {
+					http.NotFound(w, r)
+					return
+				} else {
+					http.ServeFile(w, r, fileName)
+				}
+			}))
+			defer home.Close()
+			vulnListDir := "testdata/outfiles"
+			defer os.RemoveAll(vulnListDir)
+			u := Updater{
+				vulnListDir: vulnListDir,
+				appFs:       tt.fields.appFs,
+				baseURL:     home.URL,
+				retry:       tt.fields.retry,
+			}
+			if err := u.Update(); (err != nil) != tt.wantErr {
+				t.Errorf("Update() error = %v, wantErr %v", err, tt.wantErr)
+			}
+			fileCount := 0
+			err := afero.Walk(tt.fields.appFs, vulnListDir, func(path string, info os.FileInfo, err error) error {
+				if err != nil {
+					return err
+				}
+				if info.IsDir() {
+					return nil
+				}
+				fileCount++
+
+				actual, err := afero.ReadFile(tt.fields.appFs, path)
+				assert.NoError(t, err, path)
+
+				goldenPath, ok := tt.goldenFiles[path]
+				require.True(t, ok, path)
+				expected, err := ioutil.ReadFile(goldenPath)
+				assert.NoError(t, err, goldenPath)
+				var expectedJson SaveJsonFormat
+				err = json.Unmarshal(expected, &expectedJson)
+				assert.NoError(t, err, path)
+				var actualJson SaveJsonFormat
+				err = json.Unmarshal(actual, &actualJson)
+				assert.NoError(t, err, path)
+				assert.True(t, expectedJson.PkgName == actualJson.PkgName)
+				assert.True(t, expectedJson.RepoName == actualJson.RepoName)
+				assert.True(t, expectedJson.DistroVersion == actualJson.DistroVersion)
+				for version, actualVuls := range actualJson.UnfixVersion {
+					if expVulns, ok := expectedJson.UnfixVersion[version]; !ok {
+						assert.True(t, ok)
+					} else {
+						sort.Strings(actualVuls)
+						sort.Strings(expVulns)
+						assert.True(t, reflect.DeepEqual(actualVuls, expVulns))
+					}
+				}
+				assert.True(t, expectedJson.DistroVersion == actualJson.DistroVersion)
+				require.True(t, reflect.DeepEqual(expectedJson, actualJson), "")
```

Comment:
Is there any reason that we cannot use assert.Equal?
```suggestion
				assert.Equal(t, expectedJson, actualJson)
```

Reply:
Fixed it